src/dotnet/CarbonAware.Aggregators/src/SciScore/SciScoreAggregator.cs

```diff
@@ -0,0 +1,72 @@
+using CarbonAware.Model;
+using CarbonAware.Interfaces;
+using Microsoft.Extensions.Logging;
+using System.Collections;
+using System.Globalization;
+
+namespace CarbonAware.Aggregators.SciScore
+{
+    public class SciScoreAggregator : ISciScoreAggregator
```

Comment:
I would also add xml docs on this object (assuming they aren't already on the base)